docs/fields/field-details.asciidoc

```diff
@@ -6414,6 +6430,25 @@ type: keyword
 
 
 
+| extended
+
+// ===============================================================
+
+|
+[[field-orchestrator-resource-ip]]
+<<field-orchestrator-resource-ip, orchestrator.resource.ip>>
+
+| IP address assigned to the resource being acted upon by the orchestrator. In Kubernetes, for instance, this could be the IP of a Pod (as opposed to the Node on which the Pod is running).
+
```

Comment:
minor edit suggestion:   "being acted upon by the orchestrator" -> "associated with the event being observed"

Why: events/activities in a Pod may from more than the just orchestrator - e.g. I'm kubectl exec'ing a shell in there and every time I run a command that creates some execution events.